test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js

```diff
@@ -0,0 +1,99 @@
+const userStubs = require('../../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../../stubs/gateway-account-stubs')
+
+const USER_EXTERNAL_ID = 'user-123-abc'
+const SERVICE_EXTERNAL_ID = 'service-456-def'
+const EMAIL_NOTIFICATIONS_URL = `/simplified/service/${SERVICE_EXTERNAL_ID}/account/test/settings/email-notifications`
+const GATEWAY_ACCOUNT_ID = 11
+const ADMIN_ROLE = {
+  description: 'Administrator',
+  name: 'admin',
+  permissions: [
+    {
+      description: 'Viewtransactionslist',
+      name: 'transactions:read'
+    }
+  ]
+}
+const NON_ADMIN_ROLE = {
+  description: 'View only',
+  name: 'view-only',
+  permissions: [
+    {
+      description: 'Viewtransactionslist',
+      name: 'transactions:read'
+    }
+  ]
+}
+
+const setupStubs = (role) => {
+  cy.task('setupStubs', [
+    userStubs.getUserSuccess({
+      userExternalId: USER_EXTERNAL_ID,
+      gatewayAccountId: GATEWAY_ACCOUNT_ID,
+      serviceName: { en: 'My cool service' },
+      serviceExternalId: SERVICE_EXTERNAL_ID,
+      role,
+      features: 'degatewayaccountification' // TODO remove features once simplified accounts are live
+    }),
+    gatewayAccountStubs.getAccountByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, 'test', { gateway_account_id: GATEWAY_ACCOUNT_ID })
+  ])
+}
+
+describe('email notifications settings', () => {
+  beforeEach(() => {
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+  })
+  describe('landing page', () => {
+    describe('for an admin user', () => {
+      beforeEach(() => {
+        setupStubs(ADMIN_ROLE)
+        cy.visit(EMAIL_NOTIFICATIONS_URL)
+      })
+
+      it('should show the correct heading and title', () => {
+        cy.get('h1').should('contain', 'Email notifications')
+        cy.title().should('eq', 'Settings - Email notifications')
+      })
+
+      it('should show links to change email settings', () => {
+        cy.get('.govuk-summary-list').within(() => {
+          cy.get('.govuk-summary-list__key').eq(0).should('contain', 'Collect email addresses')
+          cy.get('.govuk-summary-list__value').eq(0).should('contain', 'On (mandatory)')
+          cy.get('.govuk-summary-list__actions a').eq(0).should('contain', 'Change')
+          cy.get('.govuk-summary-list__key').eq(1).should('contain', 'Payment confirmation emails')
+          cy.get('.govuk-summary-list__value').eq(1).should('contain', 'On')
+          cy.get('.govuk-summary-list__actions a').eq(1).should('contain', 'Change')
+          cy.get('.govuk-summary-list__key').eq(2).should('contain', 'Refund emails')
+          cy.get('.govuk-summary-list__value').eq(2).should('contain', 'On')
+          cy.get('.govuk-summary-list__actions a').eq(2).should('contain', 'Change')
```

Comment:
I think we should test that these links go where they're expected to, for completeness

Reply:
They don't go anywhere at the moment as the moment I've just done the [landing page](https://github.com/alphagov/pay-selfservice/pull/4297).